### PR TITLE
DOCSP-30346: Update Kotlin Delete page

### PR DESCRIPTION
**Pull request by cbullinger**
## Pull Request Info

Update existing [Delete](https://www.mongodb.com/docs/realm/sdk/kotlin/realm-database/crud/delete/) page with missing content and updated code examples 

### Jira

- https://jira.mongodb.org/browse/DOCSP-30346

### Staged Changes

- [Delete Realm Objects - Kotlin SDK](https://preview-mongodbcbullinger.gatsbyjs.io/realm/docsp-30346-delete-page/sdk/kotlin/realm-database/crud/delete/)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/kotlin/realm-database/crud/delete.txt [hunk 1]
```diff
@@ -149,15 +252,136 @@ Delete an Embedded Object
...
+
+Remove Items from a RealmList
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can remove one or more items from a ``RealmList`` at a time:
+
+- To remove one item from the list, pass the element to 
+  `list.remove() <{+kotlin-local-prefix+}io.realm.kotlin.types/-realm-list/index.html#731697687%2FFunctions%2F878332154>`__.
+- To remove one item at a specified index in the list, pass the index to 
+  `list.removeAt() <{+kotlin-local-prefix+}io.realm.kotlin.types/-realm-list/index.html#-209439875%2FFunctions%2F878332154>`__.
```
Comment (lindseymoore) [inline source/sdk/kotlin/realm-database/crud/delete.txt:305]:
```suggestion
  `list.removeAt() <{+kotlin-local-prefix+}io.realm.kotlin.types/-realm-list/index.html#1522148962%2FFunctions%2F878332154>`__.
```
This link should link directly to removeAt! Doesn't currently.

File: source/sdk/kotlin/realm-database/crud/delete.txt [hunk 2]
```diff
@@ -149,15 +252,136 @@ Delete an Embedded Object
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can remove one or more items from a ``RealmSet`` at a time:
+
+- To remove one item from the set, pass the element 
+  you want to delete to 
+  `set.remove() <{+kotlin-local-prefix+}io.realm.kotlin.types/-realm-set/index.html#-1503494415%2FFunctions%2F-1651551339>`__.
+- To remove multiple items from the set, pass the 
+  elements you want to delete to
+  `set.removeAll() <{+kotlin-local-prefix+}io.realm.kotlin.types/-realm-set/index.html#430447804%2FFunctions%2F-1651551339>`__.
```
Comment (lindseymoore) [inline source/sdk/kotlin/realm-database/crud/delete.txt:335]: Nit: I would just be consistent with the linking to the functions. Some link directly to the entry in the table, some don't.

**Review by lindseymoore — APPROVED** [review]
LGTM! Only a few nits, great job on a long page.

File: source/sdk/kotlin/realm-database/frozen-arch.txt [hunk 3]
```diff
@@ -60,7 +66,7 @@ support change listeners.
 
 .. important:: Flows API Requires Kotlinx Coroutines
 
-   To use the Flows API in your Kotlin Multiplatform project, install the
+   To use the Flows API in your KMP project, install the
```
Comment (cbullinger) [inline source/sdk/kotlin/realm-database/frozen-arch.txt:69]:
```suggestion
   To use the Flows API in your Kotlin Multiplatform project, install the
```

**Pull request merged**